zuul.d/pipelines.yaml

```diff
@@ -689,3 +689,34 @@
       github:
         status: 'failure'
       mysql:
+
+- pipeline:
+    name: third-party-check
```

Comment:
I think we can re-use `check` pipeline at here, this config is same with `check` pipeline.